reflex-algos/components/Java/test-java-connected-multiple-jars/src/main/java/com/parallelm/components/test_java_connected_multiple_jars/JavaConnectedAddition.java

```diff
@@ -0,0 +1,8 @@
+package com.parallelm.components.test_java_connected_multiple_jars;
+
+public class JavaConnectedAddition
+{
+    public static void printHello() {
+        System.out.println("    JavaConnectedAddition: Hello");
```

Comment:
Any reason for the leading spaces in the string? (Same comment for same issue in changes that follow)